server/src/index.ts

```diff
@@ -133,6 +143,10 @@ const oauthCallback = (req: express.Request, res: express.Response, next: expres
             req.session.cookie.maxAge = expireAfterMs;
             // you have to save the session manually for POST requests like this one
             req.session.save(() => void 0);
+            if (nextPath) {
+              nextPath = undefined;
```

Comment:
why do we do this?

Reply:
@moshthepitt I think this is variable represents a store that holds one-time-use data, therefore needing to reset it after where its used. Otherwise if i failed to reset the value, then all other logins (via the auth server) would redirect to the url in nextPath irregardless of whether the user first visited that url or not, before logging in.

Reply:
@p-netm cool, thanks.  Could we leave a comment in the code that says this please?

Reply:
yeah. on it